src/utils/d3.ts

```diff
@@ -75,6 +74,43 @@ export const drawDots = <T>({
   }
 }
 
+export const drawPath = <T>({
+  svg,
+  className,
+  data,
+  x,
+  y,
+  strokeWidth,
```

Comment:
You can do
strokeWidth = 1 here
instead of `strokeWidth === undefined ? 1 : strokeWidth` in line 104